src/pe2loaddata/append_illum_cols.py

```diff
@@ -0,0 +1,114 @@
+"""append_illum_cols - Append columns corresponding to illumination functions to a LoadData .csv
+
+"""
+import argparse
+import csv
+import os
+import shutil
+import tempfile
+from os import PathLike
+from typing import Union, Any
+import yaml
+
+
+def check_file_arg(arg: Union[bytes, str, PathLike]) -> Union[bytes, str, PathLike]:
+    """Make sure the argument is a path to a file"""
+    if not os.path.isfile(arg):
+        raise argparse.ArgumentTypeError(
+            "%s is not a path to an existing file" % arg)
+    return arg
+
+
+def check_dir_arg(arg: Union[bytes, str, PathLike]) -> Union[bytes, str, PathLike]:
+    """Make sure the argument is a path to an existing directory"""
+    if not os.path.isdir(arg):
+        raise argparse.ArgumentTypeError(
+            "%s is not a path to an existing directory" % arg)
+    return arg
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="Append columns corresponding to illumination "
+                    "functions to a LoadData .csv")
+
+    parser.add_argument("--plate-id",
+                        dest="plate_id",
+                        help="Plate ID")
+
+    parser.add_argument(
+        "--illum-directory", type=check_dir_arg,
```

Comment:
Is there any reason to require the illum-directory to exist? If not, we shouldn't.

Reply:
I declare it doesn't and we shouldn't. :)